docs/dgeni.conf.js

```diff
@@ -31,12 +33,26 @@ package.config(function(log, readFilesProcessor, templateFinder, writeFilesProce
     basePath: 'src'
   }];
 
+
+  // Nunjucks and Angular conflict in their template bindings so change the Nunjucks
+  templateEngine.config.tags = {
+    variableStart: '{$',
+    variableEnd: '$}'
+  };
```

Comment:
Since nunjucks uses the same double mustache syntax for binding as angular - it is safer to change nunjucks to use `{$ ... $}` instead.